### PR TITLE
Dev/fpla 2810 - Respondent Solicitor action Events

### PR DESCRIPTION
**Pull request by imran-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2810

### Change description ###
Solicitors can now upload draft orders. 
This PR is based on code from FPLA-2809 which introduces the flattering of solicitor roles.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -1055,6 +1055,16 @@
...
+          "[SOLICITOR]",
+          "[solicitorA]",
+          "[solicitorB]",
+          "[solicitorC]",
+          "[solicitorD]",
+          "[solicitorE]",
+          "[solicitorF]",
+          "[solicitorG]",
+          "[solicitorH]",
+          "[solicitorI]",
```
Comment (Jamesnelson117) [inline ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json:1067]: @aliveni we suspect these may need to be captilised. Let me confirm again once tested on demo.

**Review by Jamesnelson117 — CHANGES_REQUESTED** [review]

File: ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json (hunk 1 above)
Comment (aliveni) [reply ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json]: @Jamesnelson117  updated the case roles and added SOLICITORJ

File: e2e/pages/events/enterRepresentativesEvent.page.js [hunk 2]
```diff
@@ -65,4 +65,16 @@ module.exports = {
...
+    I.fillField(this.fields(elementIndex).representative.email, email);
+  },
+
+  async setDigitalServingPreferences(elementIndex) {
+    I.checkOption(this.fields(elementIndex).representative.servingPreferences.digitalService);
+  },
+
+  async setRepresentativeRole(elementIndex, role) {
+    I.selectOption(this.fields(elementIndex).representative.role, role);
+  },
```
Comment (Jamesnelson117) [inline e2e/pages/events/enterRepresentativesEvent.page.js:79]: Should not need async, these methods are not doing anything asynchronous.

File: ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json [hunk 3]
```diff
@@ -318,7 +318,7 @@
     "UserRoles": ["[SOLICITOR]", "[SOLICITORA]", "[SOLICITORB]", "[SOLICITORC]", "[SOLICITORD]",
       "[SOLICITORE]", "[SOLICITORF]", "[SOLICITORG]", "[SOLICITORH]", "[SOLICITORI]", "[SOLICITORJ]"
     ],
-    "CRUD": "R"
+    "CRUD": "CRUD"
```
Comment (Jamesnelson117) [inline ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json:321]: why delete here?

Comment (aliveni) [reply ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json]: delete is not needed, removed it now

File: e2e/pages/events/enterRepresentativesEvent.page.js (hunk 2 above)
Comment (aliveni) [reply e2e/pages/events/enterRepresentativesEvent.page.js]: updated the methods and removed unused methods

**Review by Jamesnelson117 — APPROVED** [review]
LGTM

**Pull request merged**